src/features/animation/shared-view.tsx

```
@@ -154,6 +154,24 @@ export const AnimateSharedClient = ({ payload, slug }: AnimateSharedClientProps)
         href={fonts[fontFamily].src}
         crossOrigin="anonymous"
       />
+
+      {/* Aggressive letter-spacing reset for shared view */}
+      <style
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)